spec/far_mar/market_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+describe FarMar::Market do
+  before :each do
+    @market = Market.new
+  end
+
+  describe "initialize" do
+    it "initialize market"
```

Comment:
This `it` block (and each of the ones in the other three files) should have an `expect` that checks whether `@market` (or `@product`, `@sale`, etc.) is actually an instance of the appropriate class.

Check your Scrabble project spec files for an example of what we're going for.